### PR TITLE
morgan's updates on the readme file

### PR DESCRIPTION
**Pull request by m-sherrill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# practice
\ No newline at end of file
+# practice
+
+## Morgan adding some very interesting things you will want to know.......
```
Comment (mjamesd) [inline README.md:3]: omg what is it!!!

**Review by mjamesd — CHANGES_REQUESTED** [review]
I need to know the interesting things

**Review by DanPGolden — APPROVED** [review]
secrets

**Review by DanPGolden — APPROVED** [review]

**Review by yardboy2401 — APPROVED** [review]
okay

**Pull request merged**